cypress/e2e/clue/branch/teacher_tests/teacher_chat_spec.js

```diff
@@ -6,8 +6,9 @@ let dashboard = new TeacherDashboard();
 let clueCanvas = new ClueCanvas;
 let chatPanel = new ChatPanel;
 
-let selectedChatBackground = 'rgb(215, 255, 204)';
-let expandedChatBackground = 'rgb(193, 243, 179)';
+let selectedChatBackground = 'rgb(234, 242, 142)';
+let expandedChatBackground = 'rgb(234, 242, 142)';
```

Comment:
May be better to check against a class name (`comment-select`) instead of specific rgb colors

Reply:
I think I'll just leave it as RGB values since Scott was the one who wrote the original test.